main.css

```diff
@@ -0,0 +1,368 @@
+body{
```

Comment:
https://chrome.google.com/webstore/detail/perfectpixel-by-welldonec/dkaagdgjmgdmbnecmcefdhjekcoceebi?hl=en check out the site with this extension.